lib/ansible/modules/system/iptables_raw.py

```diff
@@ -0,0 +1,1040 @@
+    else:
+        iptables_new.remove_table_rule(table, name)
+
+    if keep_unmanaged:
+        iptables_new.refresh_unmanaged_rules(table)
+    else:
+        iptables_new.clear_unmanaged_rules(table)
+
+    # Refresh saved table dump with active iptables rules
+    iptables_new.refresh_saved_table_dump(table)
+
+    # Check if there are changes in iptables, and if yes load new rules
+    if iptables != iptables_new:
+
+        changed = True
+
+        # Test generated rules
+        iptables_new.system_apply_table_rules(table, test=True)
+
+        if check_mode:
+            # Create a predicted diff for check_mode.
+            # Diff will be created from rules generated from the state dictionary.
+            if hasattr(module, '_diff') and module._diff:
+                # Update unmanaged rules in the old object so the generated diff
+                # from the rules dictionaries is more accurate.
+                iptables.refresh_unmanaged_rules(table)
+                # Generate table rules from rules dictionaries.
+                table_rules_old = iptables.get_table_rules(table)
+                table_rules_new = iptables_new.get_table_rules(table)
+                # If rules generated from dicts are not equal, we generate a diff from them.
+                if table_rules_old != table_rules_new:
+                    kw['diff'] = generate_diff(table_rules_old, table_rules_new)
+                else:
+                    # TODO: Update this comment to be better.
+                    kw['diff'] = { 'prepared': "System rules were not changed (e.g. rule "\
+                                               "weight changed, redundant rule, etc)" }
+        else:
+            # We need to fetch active table dump before we apply new rules
+            # since we will need them to generate a diff.
+            table_active_rules = iptables_new.get_saved_table_dump(table)
+
+            # Apply generated rules.
+            iptables_new.system_apply_table_rules(table)
+
+            # Refresh saved table dump with active iptables rules.
+            iptables_new.refresh_saved_table_dump(table)
+
+            # Save state and system iptables rules.
+            iptables_new.system_save(backup=backup)
+
+            # Generate a diff.
+            if hasattr(module, '_diff') and module._diff:
+                table_active_rules_new = iptables_new.get_saved_table_dump(table)
+                if table_active_rules != table_active_rules_new:
+                    kw['diff'] = generate_diff(table_active_rules, table_active_rules_new)
+                else:
+                    # TODO: Update this comment to be better.
+                    kw['diff'] = { 'prepared': "System rules were not changed (e.g. rule "\
+                                               "weight changed, redundant rule, etc)" }
+
+    kw['changed'] = changed
+    module.exit_json(**kw)
+
+
+# import module snippets
+from ansible.module_utils.basic import *
```

Comment:
One question. Because I need to support Python 2.4, that means `import json` won't work, and I would have to do the import with `try/except` which is already done in `basic.py`, can I somehow import `json` from it without basically copy pasting the code used in `basic.py`?